backend/tests/TestUtils/TestUtils.fs

```diff
@@ -1261,6 +1277,7 @@ let interestingDvals : List<string * RT.Dval * RT.TypeReference> =
        [ Dval.uint16 16us ]
      ),
      TypeReference.option TUInt16)
+    // TODO: Int128 and UInt128
```

Comment:
Tried adding the following values to the `interestingDvals` List; However, the roundtripping tests failed with `Value in Datastore does not match Datastore expected type` due to the lack of database support for 128-bit ints at the moment

```
    ("option8",
     DEnum(
       Dval.optionType,
       Dval.optionType,
       Dval.ignoreAndUseEmpty [ VT.int128 ],
       "Some",
       [ Dval.int128 12Q ]
     ),
     TypeReference.option TInt128)
    ("option9",
     DEnum(
       Dval.optionType,
       Dval.optionType,
       Dval.ignoreAndUseEmpty [ VT.uint128 ],
       "Some",
       [ Dval.uint128 12Z ]
     ),
     TypeReference.option TUInt128)

```
This also required the `NumericLiteralQ`and `NumericLiteralZ`modules mentioned in this comment https://github.com/darklang/dark/pull/5139#issuecomment-1787177613